zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplineCompileTaskTest.kt

```diff
@@ -58,57 +79,19 @@ class ZiplineCompileTaskTest {
       .contains(result.task(":$taskName")!!.outcome)
   }
 
-  private fun assertZiplineCompileTask(rootProject: File, dirHasSourceMaps: Boolean) {
+  private fun assertZiplineCompileTask(rootProject: File, nonManifestFileAssertions: (ziplineFile: File) -> Unit) {
```

Comment:
What if this lambda accepted a `ZiplineFile` instead of a `File` ?
Or even better, what if this function returned a `List<ZiplineFile>` ?